### PR TITLE
Specs for Kenya Swahili Braille Code

### PR DESCRIPTION
**Pull request by JakeKyle**
Supplied to me by Martin (I will try to get his surname) from [Inclusion Resources Africa](https://inclusionresources.africa/)

### CODE REVIEW

**Review by egli — APPROVED** [review]
Looks good. You might want to add a link to the top level Readme

**Comment by JakeKyle** [comment]
Thanks @egli 
I have done this and also found out my contact is actually Martin Kieti, Chairman of Kenya Institute for the Blind, so I have added this detail to the Swahili README.

**Comment by egli** [comment]
Excellent thanks @JakeKyle

**Comment by bertfrees** [comment]
Very nice. Could you just add some more metadata like "_by_" (author) and "_published_" (data, place, by whom) please?

File: swahili/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Kenya Swahili Braille Code
+
+## [Kenya Swahili Braille Code - Graded](Kenya%20Swahili%20Braille%20Code%20-%20Graded.docx)
+
+Document supplied by Martin Kieti, Chairman of Kenya Institute for the Blind.
+
+- _applies to_: Kenya
+- _language_: English
+
+Derived from the Kenya Institute of Curriculum Development's Swahili Braille Manual: “Mwongozo wa Breli Ya Kiswahili”;
```
Comment (bertfrees) [inline swahili/README.md:12]: Do you know what exactly is the relationship between this document and the "Mwongozo wa Breli Ya Kiswahili" document?

Comment (JakeKyle) [reply swahili/README.md:12]: I understand this document is largely a translation into English of the Swahili document with a few ammendments resulting from the stakeholders' meeting. I have asked for a copy of the original Swahili document but to no avail so far. I think they someone will need to scan it.

**Pull request merged**